### ISSUE TITLE
[$500] LHN reflects wrong hint when user sents message after an attachment

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#26997]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
step 1: open workspace from user A
step 2: add user B to the work space
step 3: send attachment from user A
step 4: send message from user B
step 5: observe LHN tab hint from user A

Expected Result: LHN tab hint shows the message

Actual Result: LHN tab hint shows attachment

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.65-5
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/943e8beb-0207-4148-9f4a-cfbcbe362757

https://github.com/Expensify/App/assets/43996225/1dc0d5cb-12a1-467f-88f5-220874c198e4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @lidiyakelay
**Slack conversation:**  https://expensify.slack.com/archives/C049HHMV9SM/p1693315853678109

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0163d7b176bbb6f038</li>
        <li>Upwork Job ID: 1715323614227767296</li>
        <li>Last Price Increase: 2023-12-08</li>
    </ul>
</details>

**Comment by jscardona12** [comment]
It is reproducible on lastest main
<img width="994" alt="Screenshot 2023-09-07 at 4 42 28 PM" src="https://github.com/Expensify/App/assets/16781411/38e6c03d-dc25-4317-8d50-dd265472b593">

**Comment by s-alves10** [comment]
This would be fixed by [this PR](https://github.com/Expensify/App/pull/26850/files)
I think we can hold this issue

**Comment by johncschuster** [comment]
@allroundexperts do you agree with this?

> This would be fixed by [this PR](https://github.com/Expensify/App/pull/26850/files)
> I think we can hold this issue

**Comment by johncschuster** [comment]
Putting this on hold while we wait for https://github.com/Expensify/App/pull/26850/files

**Comment by johncschuster** [comment]
Ok! It looks like https://github.com/Expensify/App/pull/26850 has been deployed to prod. @kavimuru / @lidiyakelay can you check to see if this behavior persists?

**Comment by johncschuster** [comment]
bump @kavimuru / @lidiyakelay

**Comment by johncschuster** [comment]
bump @kavimuru / @lidiyakelay

**Comment by lidiyakelay** [comment]
@johncschuster SORRY!! for the late reply. This issue doesn't seem to be occurring anymore.

https://github.com/Expensify/App/assets/59802743/1904dce0-63ba-4197-bd80-eaceaf5a5754

**Comment by lidiyakelay** [comment]
@johncschuster but this seems to be still occurring on mobile app

**Comment by lidiyakelay** [comment]
bump @johncschuster

**Comment by lidiyakelay** [comment]
friendly bump @johncschuster

**Comment by johncschuster** [comment]
Thank you for the bump, @lidiyakelay! (And, sorry for missing them!)

Ok, it sounds like this behavior is still reproducible on mobile after #26850 was deployed. Is that about right? If so, I'll triage this out.

**Comment by s77rt** [comment]
@johncschuster Can you please remove the HOLD part from the title

**Comment by s77rt** [comment]
Not overdue. Still looking for proposals.

@johncschuster Please remove the HOLD

**Comment by johncschuster** [comment]
Oh sweet! Thanks for calling that out, @s77rt! I'll go fishing for proposals.

**Comment by samilabud** [comment]
I tried to reproduce it on mobile and it works 🤔, if anyone can reproduce it could you please send a video? Maybe I didn't try correctly 🙏🏼

**Comment by ShogunFire** [comment]
I don-t reproduce either

**Comment by s77rt** [comment]
Did you test on mobile? Based on https://github.com/Expensify/App/issues/26997#issuecomment-1759672176 and https://github.com/Expensify/App/issues/26997#issuecomment-1771675495 the bug may only be reproducible on mobile

**Comment by johncschuster** [comment]
@samilabud and @ShogunFire, did you attempt to reproduce on a physical mobile device or in a dev environment? I'm not sure if this can only be reproduced on a physical device, but it'd be good to confirm.

**Comment by samilabud** [comment]
Hi @johncschuster, confirming I have tested in the dev environment in the current version.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by s77rt** [comment]
Not overdue. Needs reproduction confirmation

**Comment by s77rt** [comment]
Same ^

**Comment by johncschuster** [comment]
@s77rt, what are your thoughts on bumping this to Weekly while we wait?

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Second week)

**Comment by s77rt** [comment]
@johncschuster I think we can just close this for now

**Comment by s77rt** [comment]
Not overdue. We can close this since it does not seem reproducible

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Third week)

**Comment by s77rt** [comment]
@johncschuster Let's close this

**Comment by s77rt** [comment]
@johncschuster ^

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Fourth week). Are we good to close?

**Comment by s77rt** [comment]
@johncschuster ^

**Comment by s77rt** [comment]
To be closed ^

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Fifth week)

**Comment by johncschuster** [comment]
Looks like we can close this one. Thanks, everyone!
